nixos/modules/installer/tools/nixos-generate-config.pl

```diff
@@ -564,10 +564,14 @@ sub multiLineList {
     ];
 
 $bootLoaderConfig
-  # networking.hostName = "nixos"; # Define your hostname.
-  # networking.wireless.enable = true;  # Enables wireless support via wpa_supplicant.
+  # Define your hostname.
+  # networking.hostName = "nixos";
+
+  # Enable wireless support via wpa_supplicant.
+  # networking.wireless.enable = true;
 
   # Select internationalisation properties.
+
```

Comment:
I think these newlines confuse it, where without the newline it is clearly tied to the section.